CHANGELOG.txt

```diff
@@ -1,3 +1,19 @@
+XXXX-YY-ZZ 1.21.x:
+------------------
+
+Enhancements:
+-------------
+
+* Whitelist the CPU_COUNT environment variable. Setting this outside conda
```

Comment:
Please shorten this and just link to the PR.  People can read more if they're interested.